muthash/nbo-java-devs
# #155738443 Launch the DetailActivity from the MainActivity

**What does this PR do?**
Creates the layout for the DetailLayout activity.
Allows DetailLayoutActivity to be launched from the MainActivity.

**Description of Task to be completed**
Launch the DetailActivity from the MainActivity

**How should this be manually tested?**
1. Clone the repository and set up the Android Studio as described on the README
2. Run the application
3. The app should run as shown on the video below

**What are the relevant pivotal tracker stories?**
- 155820645
- 155820624

**Screenshot**
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/33781312/43891442-f6ae87e4-9bd1-11e8-9051-f029ad795808.gif)
